### DISCUSSION TITLE
Github Container registry questions

### DISCUSSION THREAD

**Discussion by surajkota** [community/community#27387] [Actions]
Hi, I have a few questions about GitHub Container registry:

1. Availability - What is the availability SLA for Github Container registry and it is publically documented?
2. Rate limiting - I could not find information about rate limiting for unauthenticated container pulls. Are there any such limits? Or plans to add if there are none today?
3. Pricing - [This page](https://docs.github.com/en/billing/managing-billing-for-github-packages/about-billing-for-github-packages) mentions github packages is free for public repositories. Does that mean there is unlimited storage and no charges for data transfer in and out for public container registries? The reason I ask this is because there is a note on the site:

> Billing update for container image storage: The period of free use for container image storage and bandwidth for the Container registry has been extended. If you are using Container registry you'll be informed at least one month in advance of billing commencing and you'll be given an estimate of how much you should expect to pay.

Is the above note relevant for public repositories as well?

**Comment by riverma** [comment]
+1. Also interested in getting clarification on the above questions for public repositories.
